contrib/prosody/files/prosody

```diff
@@ -0,0 +1,8 @@
+# prosody system service
+
+type = scripted
```

Comment:
make a real process service

Reply:
i wonder if reloading will work with it via signal